src/frontend/components/SøknadsSteg/VelgBarn/LeggTilBarn/useLeggTilBarn.tsx

```diff
@@ -114,7 +108,7 @@ export const useLeggTilBarn = (): {
                     {
                         ident: ident.verdi,
                         borMedSøker: undefined,
-                        fødselsdato: [fulltÅr, måned, dagKompansertForDNummer].join('-'),
+                        fødselsdato: fødselsdatoSomISOStringFraIdNummer(ident.verdi),
```

Comment:
FYI man kan ikke stole på fnr til fødselsdato. Det er folk som har annen fødselsdato enn det som står i fnr :(

Reply:
Vi veit om spesialtilfeller hva gjelder D-nummer, er det spesialtilfeller for fødselsnummer også? Skal vi gjøre oppslag mot PDL for å hente alder liksom?

Reply:
Ja, vi bruker aldri fnr til dato i sak. Henter alltid fødselsdato :)

Reply:
Problemet da er vel at hvis vi gjør det samme her har vi har laga et input-felt som er offentlig tilgjengelig der man kan brute-force fødselsnummer, og det er vel litt dumt?

Reply:
Ja, vi kan ikke lage søk-funksjonalitet i søknadsdialoger. Jeg ville bare gi beskjed om at fnr ikke kan benyttes til å vise fødselsdato. Så da bør man vurdere å ikke vise fødselsdato eller la bruker fylle det inn tror jeg.

Reply:
Da blir det sånn, ingen alder på dem :) Synd da, vi som hadde tjukt med tester for denne :smiley: